dev-requirements.txt

```
@@ -4,6 +4,7 @@ isort; python_version < "3.8"
 isort==5.13.2; python_version >= "3.8"
 pre-commit; python_version <= "3.8"
 pre-commit==4.0.1; python_version > "3.8"
+codespell
```

```suggestion
codespell=1.16.0
```

Explicitly assigning the same version as pre-commit installs just to make life easier for devs.  We have a script that updates both places monthly.